ices/86086.rs

```diff
@@ -0,0 +1,2 @@
+trait Tr : Sized { async extern r###"��áÖ@"### fn union < > ( # [ ']' '[' 'default' 'async' '&raw' ] & '?ဪൈ self , ... , ... , ... , ... , ... , ... , ... , ... , ... , ) -> && '_ mut ( ) ;
```

Comment:
Note: This and some other issues are duplicates of the existing one but I'm going to include for now. We can drop them anytime.